game/client_types/autoplay.js

```diff
@@ -10,11 +10,13 @@
 
 module.exports = function(treatmentName, settings, stager, setup, gameRoom) {
 
-    const channel = gameRoom.channel;
-    const node = gameRoom.node;
+    let channel = gameRoom.channel;
+    let node = gameRoom.node;
     const ngc = require('nodegame-client');
 
-    let game, stager;
+    let game;
+
+    var stager;
```

Comment:
No need to define variable `stager` it is already defined in the function definition above. Just remove the `var stager` line.

Generally, `let` should be used right where the variable is assigned a value, unless you have an if statement assigning it. Can you do `let game = ...`

Reply:
ok, done!